src/database/sql_database.cc

```diff
@@ -1241,7 +1241,9 @@ int SQLDatabase::ensurePathExistence(const fs::path& path, int* changedContainer
     if (changedContainer && *changedContainer == INVALID_OBJECT_ID)
         *changedContainer = parentID;
 
-    return createContainer(parentID, f2i->convert(path.filename()), path, OBJECT_FLAG_RESTRICTED, false, "", INVALID_OBJECT_ID, std::vector<std::pair<std::string, std::string>>());
+    std::vector<std::pair<std::string, std::string>> itemMetadata;
+    itemMetadata.emplace_back(std::make_pair(MetadataHandler::getMetaFieldName(M_DATE), to_dcDate(fs::last_write_time(path))));
```

Comment:
you can use `to_seconds()` on `last_wirte_time` and then format it
`fmt::format("{:%FT%T%z}", fmt::localtime(to_seconds(fs::last_write_time(path)).count()))`